### DISCUSSION TITLE
Free "Teams" tier for nonprofit research GitHub organisations

### DISCUSSION THREAD

**Discussion by bobturneruk** [community/community#57803] [GitHub Education] [answered]
### Hi everyone,

Hi! My reading of the benefits for individual "teacher" / faculty members of GitHub Education is that they should be able to enable free "Teams" tier features for GitHub organisations doing nonprofit research (https://education.github.com/benefits). However, when I try to do this I'm prompted for billing information at the standard price per user. Can anybody advise on this, please? Thanks for reading!

**Comment by manathan1984** [comment]
Same exact problem #57451 -- any luck?

**Reply by bobturneruk** [reply]
No - sorry to say I've not heard anything. (And sorry I missed your post here and didn't add to it, but made a new one.) Given that you've been redirected from support to community on this, and waited some time, it would be great if folks reading this who are interested in the answer engage with the thread by upvoting, or by making a comment. Hopefully this will draw attention to the question.

**Comment by bobturneruk** [comment, marked as answer]
The excellent @Robadob has found this https://education.github.com/globalcampus/teacher - I'm checking if it works on a test organisation.

**Reply by bobturneruk** [reply]
We can see the status of applications here (thanks again @robadob) https://education.github.com/discount_requests/application

**Reply by mwt** [reply]
I've used it before. It works. It just sets the price per seat to zero.

**Reply by bobturneruk** [reply]
Thanks @mwt! It's taking some time for that to happen for my test organisation.

![image](https://github.com/community/community/assets/34244196/a1a0dcfb-2e37-4866-91cf-8716a01587d2)

I'll update when I'm properly able to upgrade the organisation.

**Reply by bobturneruk** [reply]
Still not upgraded, and still asks for payment if I click through to upgrade.

**Reply by bobturneruk** [reply]
It's now been upgraded (I got an email). Took about 3 days.

**Comment by mmoayyed** [comment]
What is the general timeline for GitHub to respond to non-profit requests? My request has not received any replies since mid-April 2023.

**Reply by bobturneruk** [reply]
I don't know. I've only used it for academic stuff.

**Reply by mwt** [reply]
I've done the nonprofit one, but it was instant and automatic in my case. So, it's still not very helpful.
